frontend/src/lib/components/Cards/InsightCard/InsightCard.tsx

```diff
@@ -68,6 +69,10 @@ export interface InsightCardProps extends Resizeable {
     doNotLoad?: boolean
     /** Dashboard variables to override the ones in the insight */
     variablesOverride?: Record<string, HogQLVariable>
+    /** Dashboard breakdown colors to override the ones in the insight */
+    breakdownColorOverride?: BreakdownColorConfig[]
+    /** Dashboard color theme to override the ones in the insight */
+    dataColorThemeId?: number | null
```

Comment:
We don't actually use these props, but use them in `DashboardItems` to force a re-render. It wasn't possible to use kea to achieve the same thing, as the `InsightCard` is also used in a context where `dashboardLogic` isn't available and "dynamic selectors" like `dashboardLogic(props).values.someValue` aren't notified of updates